### PR TITLE
MODINVOICE-53- Adjustment schema changes

### PR DESCRIPTION
**Pull request by KVupp**
## Purpose
[MODINVOICE-53](https://issues.folio.org/browse/MODINVOICE-53) - Adjustment schema changes

## Approach
1. Modify test data

#### TODOS and Open Questions
- [x] Merge [acq-models](https://github.com/folio-org/acq-models/pull/136) first and then point this branch to latest master
- [ ] Merge same time with https://github.com/folio-org/mod-invoice-storage/pull/20

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [x] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/test/resources/mockdata/invoices/invoices.json [hunk 1]
```diff
@@ -3,6 +3,14 @@
     {
       "id": "c0d08448-347b-418a-8c2f-5fb50248d67e",
       "approvalDate": "2018-07-29T00:00:00.000+0000",
+      "adjustments": [
+       {
+      	"description": "Extra Taxes",
```
Comment (craigmcnally) [inline src/test/resources/mockdata/invoices/invoices.json]: indentation

File: src/test/resources/mockdata/invoices/invoice.json [hunk 2]
```diff
@@ -2,6 +2,14 @@
   "id": "c0d08448-347b-418a-8c2f-5fb50248d67e",
   "approvedBy": "ab18897b-0e40-4f31-896b-9c9adc979a88",
   "approvalDate": "2018-07-29T00:00:00.000+0000",
+  "adjustments": [
+    {
+      "description": "Extra Taxes",
+      "type": "Amount",
+      "value": 10
```
Comment (craigmcnally) [inline src/test/resources/mockdata/invoices/invoice.json]: shouldn't this have `"prorated": "Not prorated"`

File: src/test/resources/mockdata/invoiceLines/invoice_lines.json [hunk 3]
```diff
@@ -64,9 +64,9 @@
       ],
       "adjustments": [
         {
-          "adjustmentLabel": "Adjustment label",
-          "adjustmentType": "Percentage",
-          "adjustmentValue": 10.1
+          "description": "Adjustment label",
+          "type": "Percentage",
+          "value": 10.1
```
Comment (craigmcnally) [inline src/test/resources/mockdata/invoiceLines/invoice_lines.json]: I think these also need the `prorate` field as it's required

Comment (KVupp) [reply src/test/resources/mockdata/invoiceLines/invoice_lines.json]: Done

File: src/test/resources/mockdata/invoices/invoice.json (hunk 2 above)
Comment (KVupp) [reply src/test/resources/mockdata/invoices/invoice.json]: correct. Added the relationToTotal as well

**Pull request merged**